3.4/arm_alpine/start.sh

```diff
@@ -42,6 +42,8 @@ then
   echo "env[MOODOLE_DB_PORT] = '$MOODOLE_DB_PORT'" >> $PHP_FM_CONFIG
 fi
 
+envsubst '$NGINX_PORT' < $NGINX_CONFIG.template > $NGINX_CONFIG
```

Comment:
you should put it in the top in every version because we need to render the template before adding manipulation